### ISSUE TITLE
[a11y-defect-3]: Download benefit letters "Expand all" button should not have an `aria-expanded` attribute

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#96354]
### Point of contact

Trevor Pierce

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

The VA Documents and Letters (step 2) view has an "Expand all" button for the accordions. This button has an `aria-expanded` attribute that doesn't make sense here because it's a control for multiple accordions, with text that updates to match its current state. Screenshot attached.

---
![Image](https://github.com/user-attachments/assets/8aaaf4ca-92a7-41b9-8158-10a07707037e)

### Reproduction steps

1. Log into https://staging.va.gov/records/download-va-letters/letters/letter-list with a test user who has benefit letters. I used user 54.
2. Turn on the screen reader of your choice
3. Tab to the "Expand all" button and listen for it to say "Expand all accordions, button, collapsed"
4. Press `Enter` to expand the accordions
5. Verify the button now says "Close all accordions, button, expanded"

### Proposed solution or next steps

I'm proposing we remove the `aria-expanded` attribute from this button and just use the text to describe what action the button will take.

### References, articles, or WCAG support

1. Understanding [SC 1.3.1](https://w3.org/TR/WCAG22#info-and-relationships): Info and Relationships (Level A)

### Type of issue

- [ ] Axe-core
- [x] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [x] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by 1Copenut** [comment]
@dysbo I'm grabbing this one off the backlog while I'm blocked on my other accessibility ticket. Should be a very quick win for us.

**Comment by 1Copenut** [comment]
Turns out this is an upstream item in the VADS. I'll pivot and work in that repo to issue a patch fix.

**Comment by dysbo** [comment]
Moved this one to blocked because Trevor needs JAWS in order to test the change completely and we are currently navigating that.
